### ISSUE TITLE
migration priorityqueue to k3priorityqueue

### ISSUE THREAD

**Issue by pengsven** [drmingdrmer/pykit3-super#5]
### what do you do

- migration priorityqueue to k3priorityqueue

### reference

- https://github.com/pykit3/pk3

**Comment by drmingdrmer** [comment]
It depends on `heap` 🤔 ...

**Comment by IloveKanade** [comment]
> It depends on `heap` 🤔 ...

yep, i told sevn. so, the priority queue and heap will be migrated together

**Comment by drmingdrmer** [comment]
> > It depends on `heap` 🤔 ...
> 
> yep, i told sevn. so, the priority queue and heap will be migrated together

What about one by one?
Since these are actually two packages...

**Comment by IloveKanade** [comment]
> > > It depends on `heap` 🤔 ...
> > 
> > 
> > yep, i told sevn. so, the priority queue and heap will be migrated together
> 
> What about one by one?
> Since these are actually two packages...

I mean I do the heap migration first and then the priority queue. This should work, right?

**Comment by drmingdrmer** [comment]
> I mean I do the heap migration first and then the priority queue. This should work, right?

Perfect!

**Comment by IloveKanade** [comment]
> > I mean I do the heap migration first and then the priority queue. This should work, right?
> 
> Perfect!

ok,let's do this🤪
